### PR TITLE
Added Alerts and Badges

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Alerts and Badges in MADEUI.

### CODE REVIEW

File: components/Alerts/Alerts.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    color:rgb(41,78,168);
+}
+.sucess__alert{
+    background-color: rgb(132,241,201);
+    color:rgb(27,112,82);
+}
+.danger__alert{
+    background-color: rgb(252,197,197);
+    color:rgb(220,53,69);
+
```
Comment (MiheerTamkhane) [inline components/Alerts/Alerts.css:21]: Creating variables for color should be helpful.

File: components/Alerts/Alerts.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    <link rel="stylesheet" href="/Pages/Pages.css">
+    <link rel="stylesheet" href="/components/Alerts/Alerts.css">
+    <script src="https://kit.fontawesome.com/fda6e23714.js" crossorigin="anonymous"></script>
+    <title>MADEUI</title>
+</head>
+<body>
+    <div class="Pages">
+       
+            
+        <header class="header Pages_header" id="header">
```
Comment (MiheerTamkhane) [inline components/Alerts/Alerts.html:17]: I think you should avoid class names with capital letters..

**Review by MiheerTamkhane — APPROVED** [review]
Looks good to me.

**Review by pritam-kr — APPROVED** [review]

**Review by srishti-maurya — APPROVED** [review]

File: components/Alerts/Alerts.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply components/Alerts/Alerts.css:21]: Okay..will work on it...thanks.

**Pull request merged**